{{cookiecutter.project_slug}}/src/{{cookiecutter.package_name}}/rest_config.py

```diff
@@ -7,7 +7,7 @@
 
 from servicelib.application_keys import APP_OPENAPI_SPECS_KEY
 
-APP_OPENAPI_SPECS_KEY = APP_OPENAPI_SPECS_KEY
+APP_OPENAPI_SPECS_KEY = APP_OPENAPI_SPECS_KEY    # pylint: disable=self-assigning-variable,bad-option-value
```

Comment:
there is no need anymore for this ``a = a``. This is old ... OK, we will clean this up in another PR

Reply:
great!

Reply:
If this gets fixed, we can downgrade pylint to the previous versions